backend/MilkList/query.py

```diff
@@ -1 +1,35 @@
-## reserve for milklist
+import graphene
+import json
+
+class MilkOptionType(graphene.ObjectType):
+    name = graphene.String()
+
+class MilkKeyType(graphene.ObjectType):
+    milk_1 = graphene.Field(MilkOptionType)
+    milk_2 = graphene.Field(MilkOptionType)
+    milk_3 = graphene.Field(MilkOptionType)
+    milk_4 = graphene.Field(MilkOptionType)
+    milk_5 = graphene.Field(MilkOptionType)
```

Comment:
Why do you need to hard code Milk 1,2,3

Reply:
Will your graphql breaks if we add/remove additional milk?

Reply:
yes, i will make the helper function to deal with it
i was just happy that it works, cuz i been working on this problem for a while LOL

Reply:
updated with helper function as well as to make sure it create dynamically 
d35e50814b99f38e1f4b6fb0a013fde871a352d2